src/js/test/index.test.js

```diff
@@ -86,3 +86,144 @@ describe('currentScreenWidth', () => {
         expect(width).toBe(false);
     });
 });
+
+describe('withinBreakpoint', () => {
+    it('should return true if I pass in `>narrow` when my screen width is larger than 414', () => {
+        // Arrange
+        TestUtils.setBodyHtml(`
```

Comment:
Extract the `setBodyHtml` function call out into a `beforeEach` as it is repeated in every test below?